components/finance/ExpenseButton/index.tsx

```diff
@@ -0,0 +1,70 @@
+import { Button, createStyles, makeStyles } from '@material-ui/core'
+import React, { useState } from 'react'
+import { useToggle } from 'react-use'
+
+import ClientSideRendering from '../../../lib/client-side-rendering'
+import ExpensePanelDetails from '../ExpensePanelDetails'
+
+const useStyles = makeStyles(() =>
+  createStyles({
+    deleteButton: {
+      color: 'white',
+      background: '#f16c6c',
+      '&:hover': {
+        background: '#f16c6c',
+      },
+    },
+  })
+)
+
+export default function ExpenseButton() {
+  const classes = useStyles()
+  const [isOn, toggleIsOn] = useToggle(false)
+  const [amount, updateAmountSum] = useState(0.0)
+  const currencyList = ['INR', '$', 'YUH']
+  const defaultCurrency = '$'
+  const [totalValues, setTotalValues] = useState({
+    rows: [
+      {
+        currency: defaultCurrency,
+        amount: 0,
+        comment: '',
+        tags: '',
+        person: '',
+      },
+    ],
+  })
+
+  return (
+    <ClientSideRendering>
+      <div className="ml-24">
+        <Button
+          size="large"
+          type="submit"
+          color="primary"
+          className={classes.deleteButton}
+          onClick={toggleIsOn}
+          onKeyPress={toggleIsOn}
+        >
+          <div className="text-sm">{defaultCurrency}</div>
+          <div className="text-lg">{amount.toString().split('.')[0]}</div>
+          <div className="text-sm">
+            {amount.toString().split('.').length > 1 ? `.${amount.toString().split('.')[1].substr(0, 2)}` : '.00'}
```

Comment:
this logic should move into a method within this component and not written in the render method (for readability)